### ISSUE TITLE
Something about this file isn't highlighted correctly

### ISSUE THREAD

**Issue by NullVoxPopuli** [NullVoxPopuli/highlightjs-glimmer#350]
![image](https://github.com/NullVoxPopuli/highlightjs-glimmer/assets/199018/109643fa-76f4-4e86-b2be-370fb377dc08)

it seems to occur with any combination of line-above-the-template-removals.

Only when there are no lines above the template tag does it all get highlighted.

```gjs
import { on } from '@ember/modifier';
import { cell } from 'ember-resources';
import { PortalTargets, /* Portal, PORTALS */ } from 'ember-primitives';

const isOpen = cell(false);

<template>
  {{! Application-level targets }}
  <PortalTargets />

  <button {{on 'click' isOpen.toggle}}>
    Toggle
    <Tooltip> {{! uses <Portal @to={{PORTALS.tooltip}}> }}
      Toggles the modal state
    </Tooltip>
  </button>

  {{#if isOpen.current}}
    <Modal> {{! uses <Portal @to={{PORTALS.modal}}> }}
      <:body>
        <PortalTargets />

        <p>
          this is some text
          <Tooltip> {{! uses <Portal @to={{PORTALS.tooltip}}> }}
            This is still positioned above the Modal
            because the nearest portal targets are within the modal.
          </Tooltip>
        </p>
      </:body>
    </Modal>
  {{/if}}

</template>
```

**Comment by NullVoxPopuli** [comment]
oh that's a big win! excellent! thank you!

**Comment by NullVoxPopuli** [comment]
and now released in 2.2.0: https://github.com/NullVoxPopuli/highlightjs-glimmer/releases/tag/highlightjs-glimmer%402.2.0
